src/main/typescript/node_modules/@atomist/rug/operations/Handlers.ts

```diff
@@ -23,7 +23,7 @@ interface Instruction<T extends InstructionKind> {
 }
 
 class Respondable<T extends InstructionKind> {
-  instruction: Instruction<T>
+  instruction: Review | Edit | Generate
```

Comment:
What about an 'execute'? We need to be able to respond to those.

Also, we still need to be able to add non-respondable instructions to the Plan.